tools/python/convert_from_tensorflow.py

```diff
@@ -249,6 +249,22 @@ def dump_maximum_to_file(self, node, f):
         np.array([input_operand_index, output_operand_index], dtype=np.uint32).tofile(f)
 
 
+    def dump_reshape_to_file(self, node, f):
+        assert(node.op == 'Reshape')
+        self.layer_number = self.layer_number + 1
+        rnode = self.name_node_dict[node.input[1]]
+        new_numbers = rnode.attr['value'].tensor.tensor_content[0]
+        new_height = rnode.attr['value'].tensor.tensor_content[4]
+        new_weight = rnode.attr['value'].tensor.tensor_content[8]
+        new_channels = rnode.attr['value'].tensor.tensor_content[12]
```

Comment:
should be tensor_content[0:4]   tensor_content[4:8], ...

Reply:
i stepped into this and found new_height is not -1.  Should use struct.unpack for a conversion.